src/data_management/timeseries/locations/EditLocation.tsx

```diff
@@ -12,6 +12,9 @@ interface RouteProps {
 const EditLocation = (props: RouteProps & RouteComponentProps) => {
   const organisations = useSelector(getOrganisations);
   const [currentRecord, setCurrentRecord] = useState(null);
+  const [relatedAssetRequired, setRelatedAssetRequired] = useState(true);
+  const [relatedAsset, setRelatedAsset] = useState(null);
+  // Todo fix this ts ignore
   // @ts-ignore
   const { uuid } = props.match.params;
```

Comment:
if you go to for example: EditTimeseries.tsx component, I added a type definition for RouteProps which is wrapped inside RouteComponentProps type: `RouteComponentProps<RouteProps>`. In the RouteProps, you can define the type for `props.match.params`.

Reply:
thanks !